event_test.go

```diff
@@ -83,7 +83,10 @@ func TestUnmarshalEvent(t *testing.T) {
 }
 
 func TestUnmarshalEvents(t *testing.T) {
-	origRegisteredEvents := registeredEvents
+	origRegisteredEvents := map[string]Event{}
```

Comment:
Thanks for reviewing @neelance, good call on the shorter/[performant](https://groups.google.com/forum/#!msg/golang-nuts/UvUm3LA1u8g/jGv_FobNpN0J) approach to clear map; commit 70d8733e8273884ad6631df840c36f22df3a82bf includes the change.